content-src/components/DiscoveryStreamComponents/DSTextPromo/_DSTextPromo.scss

```diff
@@ -1,18 +1,27 @@
 .ds-dismiss-ds-text-promo {
-  width: 744px;
```

Comment:
If you add this back in, but change it from `width: 744px;` to `max-width: 744-x;` You can get both the old size at max breakpoint, and the smaller breakpoints work how you have them. Thoughts?